mlflow/tracing/processor/mlflow.py

```diff
@@ -86,10 +86,26 @@ def on_start(self, span: OTelSpan, parent_context: Optional[Context] = None):
         span.set_attribute(SpanAttributeKey.REQUEST_ID, json.dumps(request_id))
 
     def _start_trace(self, span: OTelSpan, start_time_ns: Optional[int]) -> TraceInfo:
+        from mlflow.tracking.fluent import _active_run_stack
+
         experiment_id = get_otel_attribute(span, SpanAttributeKey.EXPERIMENT_ID)
         metadata = {TRACE_SCHEMA_VERSION_KEY: str(TRACE_SCHEMA_VERSION)}
         # If the span is started within an active MLflow run, we should record it as a trace tag
-        if run := mlflow.active_run():
+        # Note `MLflow.active_run()` can only get thread-local active run,
+        # but tracing routine might be applied to model inference worker threads
+        # (e.g. langchain model batch inference),
```

Comment:
Does "batch inference" here mean `chain.batch()` in LangChain or our `predict` method (api_request_parallel_processor)?

Reply:
it means both of the cases you listed above

Reply:
Could you list two explicitly?